atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/transaction/TransactionConfig.java

```diff
@@ -51,6 +52,15 @@ public boolean lockImmutableTsOnReadOnlyTransactions() {
         return false;
     }
 
+    /**
+     * Indicates how long user transactions are allowed to take to commit, in terms of how long we'll refresh the
+     * commit locks for.
+     */
+    @Value.Default
+    public HumanReadableDuration commitLockTenure() {
+        return HumanReadableDuration.minutes(5);
```

Comment:
Are we sure this default is ok for all users of atlasdb?

Reply:
It is potentially a bit strict, but for the most part should be fine. What do you think would be good?

Reply:
My main worry is if someone runs a huge serializable read-only transaction

Reply:
Exactly, I think 5 minutes should be plenty for 99% of the cases, but there may be outliers